### PR TITLE
feat(suite-native): production workflow

### PR DESCRIPTION
**Pull request by juriczech**
## Description
Enable EAS to build and submit production build on iOS and Android. 

Had to create a plugin to remove push notification ios settings, tweak some stuff in `app.config.ts`.

## Related Issue
https://github.com/trezor/trezor-suite/issues/10789

### CODE REVIEW

File: .github/workflows/suite-native_production_ci.yml [hunk 1]
```diff
@@ -1,23 +1,44 @@
...
+on:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: Install and build
```
Comment (vdovhanych) [inline .github/workflows/suite-native_production_ci.yml:12]: This should also be creating environment release for release approvement to work
```yaml
  build:
    name: Install and build
    environment: production
```
cc @matejkriz

**Review by vdovhanych — CHANGES_REQUESTED** [review]

File: suite-native/app/app.config.ts [hunk 2]
```diff
@@ -99,6 +101,7 @@ export default ({ config }: ConfigContext): ExpoConfig => {
                         },
                     },
                 },
+                UIRequiredDeviceCapabilities: ['armv7'],
```
Comment (matejkriz) [inline suite-native/app/app.config.ts:104]: Why this change?

Comment (juriczech) [reply suite-native/app/app.config.ts:104]: This was missing from expo migration and was causing error in submission to the app store connect. Basically app store's complaint was that we're changing device compatibility. https://expo.dev/accounts/trezorcompany/projects/trezor-suite/submissions/1e284324-933c-4f15-83df-373e6d8808dd
image

Comment (matejkriz) [reply suite-native/app/app.config.ts:104]: thanks 👌

File: .github/workflows/suite-native_production_ci.yml (hunk 1 above)
Comment (juriczech) [reply .github/workflows/suite-native_production_ci.yml:12]: cc99ee1f2041b704239b27d05f9fbb69f77eee57

**Review by matejkriz — APPROVED** [review]

**Review by vdovhanych — APPROVED** [review]

File: suite-native/app/eas.json [hunk 3]
```diff
@@ -32,10 +32,6 @@
     },
     "submit": {
         "develop": {
-            "android": {
```
Comment (Nodonisko) [inline suite-native/app/eas.json:35]: Why?

Comment (juriczech) [reply suite-native/app/eas.json:35]: It's not used since we deploy to firebase by a shell script.
